.github/workflows/deploy_production.yml

```
@@ -0,0 +1,92 @@
+name: Deploy Translations Production
+
+on:
+    # Run this workflow on push to production-release tag (via chatops)
+    push:
+        tags:
+          - production-release
+
+    # Allow running this workflow manually from the Actions tab
+    workflow_dispatch:
+
+jobs:
+  deploy_staging_branch:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: azure/login@v1
+      with:
+          creds: ${{ secrets.AZURE_STATIC_SITES }}
+
+    - name: Node.js build
+      id: build
+      uses: actions/setup-node@v1
+      with:
+        node-version: '14.x'
+    - run: npm install
+    - run: npm run _build-production
+
+    - name: Upload to blob storage
+      uses: azure/CLI@v1
+      with:
+        creds: ${{ secrets.AZURE_STATIC_SITES }}
+        inlineScript: |
+            az storage blob upload-batch \
+              --account-name zooniversestatic \
+              --content-cache-control 'public, immutable, max-age=604800' \
+              --destination '$web/alice.zooniverse.org' \
```

Still pointed to alice here.
```suggestion
              --destination '$web/translations.zooniverse.org' \
```